limits.c

```diff
@@ -77,6 +77,7 @@ void limits_enable(uint8_t axes, uint8_t expected) {
   limits.expected = bit_istrue(settings.flags,BITFLAG_INVERT_LIMIT_PINS)?~expected:expected;
   limits.active = axes<<LIMIT_BIT_SHIFT;
   limits.mag_gap_check = MAG_GAP_CHECK_ENABLE;
+  memcpy(sys.probe_position, sys.position, sizeof(float) * N_AXIS);
```

Comment:
This seems out of place.  Why are we dealing with the probe_position in this function?